### PR TITLE
New filter to visualizations listing: Liked datasets/maps

### LINKED ISSUE #1801: Liked datasets/maps filter is not working properly
If says there is only one liked dataset but it keeps returning all datasets.

screen shot 2015-01-15 at 00 37 55

### PR DESCRIPTION
**Pull request by Kartones**
Closes #1801

### CODE REVIEW

**Comment by Kartones** [comment]
@rafatower CR please

File: lib/assets/javascripts/cartodb/new_dashboard/main_view.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ module.exports = cdb.core.View.extend({
       only_shared:    params.shared,
       exclude_shared: !params.shared,
       locked:         params.locked,
-      liked:          params.liked,
+      only_liked:     params.liked,
       order:          order,
       type:           params.content_type === "datasets" ? 'table' : 'derived'
```
Comment (rafatower) [inline lib/assets/javascripts/cartodb/new_dashboard/main_view.js]: is the `content_type` always set?

Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_dashboard/main_view.js]: Yes, always has a value.

**Comment by rafatower** [comment]
:+1:

**Pull request merged**